specification/sql/resource-manager/Microsoft.Sql/preview/2022-08-01-preview/Servers.json

```diff
@@ -931,14 +931,15 @@
           "type": "string"
         },
         "publicNetworkAccess": {
-          "description": "Whether or not public endpoint access is allowed for this server.  Value is optional but if passed in, must be 'Enabled' or 'Disabled'",
+          "description": "Whether or not public endpoint access is allowed for this server.  Value is optional but if passed in, must be 'Enabled' or 'Disabled' or 'SecuredByPerimeter'",
           "enum": [
             "Enabled",
-            "Disabled"
+            "Disabled",
+            "SecuredByPerimeter"
           ],
           "type": "string",
           "x-ms-enum": {
-            "name": "ServerNetworkAccessFlag",
+            "name": "ServerPublicNetworkAccessFlag",
```

Comment:
>ServerPublicNetworkAccessFlag [](http://example.com/codeflow?start=21&length=29)

Why call it a flag? [As a C# user I get confused. Its not like a flags enum is it.]

Reply:
I never touched any of this. Not sure why swagger generator is picking up the changes.

Reply:
Its probably not, I don't recall exactly why I happened to look at it while reviewing.